src/lib.rs

```diff
@@ -62,6 +67,16 @@ pub enum Recipe {
   Command(Command),
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Include {
+  /// Git URL of a remote repo
+  pub url: String,
+
+  /// Git ref to keep up with
+  #[serde(alias = "ref", default = "default_git_ref")]
+  pub ref_: String,
+}
```

Comment:
This should have a `file` field like `Group`, no? Looks like right now it's just using the default set of moldfile names for discovery, but I can imagine some scenario where one repo might have multiple moldfiles. eg, a Python mold that might have `pipenv` and `poetry` based moldfiles.